source/reviews/speed.txt

```diff
@@ -0,0 +1,149 @@
+----------------------
+
+There is one time where the consideration of personal velocity trumps team
+velocity. **If you are in the middle of a focused task, such as writing content,
+don't interrupt yourself to do a code review.**
+Research has shown that it can take a long time for an author to get back into a
+smooth flow of work after being interrupted. Interrupting yourself while writing
+is actually *more* expensive to the team than making another author wait a bit
+for a review.
+
+Instead, wait for a break point in your work before you respond to a request for
+review. This could be when your current task is completed, after lunch,
+returning from a meeting, coming back from the restroom, etc.
+
+Fast Responses
+--------------
+
+When we talk about the speed of reviews, it is the *response* time that we
+are concerned with, as opposed to how long it takes a PR to get through the
+whole review and be accepted. The whole process should also be fast, ideally,
+but it's even more important for the *individual responses* to come quickly
+than it is for the whole process to happen rapidly.
+
+Even if it sometimes takes a long time to get through the entire review
+*process*, having quick responses from the reviewer throughout the process
+significantly eases the frustration authors can feel with "slow" reviews.
+
+If you are too busy to do a full review on a PR when it comes in, you can still
+send a quick response that lets the author know when you will get to it,
+suggest other reviewers who might be able to respond more quickly, or
+provide some initial broad comments. (Note: none of this means
+you should interrupt focus work to send a response; send the response at
+a reasonable break point in your work.)
+
+**It is important that reviewers spend enough time on review that they are
+certain their "LGTM" means "this code meets :ref:`our standards <pr-standard>`"**
+However, individual responses should still fast.
+
+Cross-Time-Zone Reviews
+-----------------------
+
+When dealing with time zone differences, try to get back to the author when they
+are still in their working hours. If they have already gone offline, then
+try to make sure your review is done before they get back online the next day.
+
+LGTM With Comments
+------------------
+
+In order to speed up reviews, there are certain situations in which a reviewer
+should give LGTM/Approval even though they are also leaving unresolved
+comments on the PR. This is done when either:
+
+- The reviewer is confident that the author will appropriately address all
+  the reviewer's remaining comments.
+- The remaining changes are minor and don't *have* to be done by the
+  author.
+
+The reviewer should specify which of these options they intend if it is not
+otherwise clear.
+
+LGTM With Comments is especially worth considering when the author and
+reviewer are in different time zones and otherwise the author would be
+waiting for a whole day just to get "LGTM"
+
+Large PRs
+
```

Comment:
As the author, your responsibility should be to point the reviewer to and describe all the relevant changes you've made in a complex or large PR.